xmipp

```diff
@@ -358,28 +361,12 @@ def getAllBranchesAndTags(repo):
                     show_output=False, log=log, show_command=False)
     if result:
         branchesAndTags = [l.split('/')[-1] for l in log]
-        return True, branchesAndTags
+        return branchesAndTags
```

Comment:
instead of:
```
branchesAndTags = [l.split('/')[-1] for l in log]
return branchesAndTags
```
Maybe simpler just:
```
return [l.split('/')[-1] for l in log]
```

Reply:
maybe but more readable

Reply:
Then maybe:
```Python
# Returning list containing branches and tags
return [l.split('/')[-1] for l in log]
```